graphql/schema.lua

```diff
@@ -117,7 +210,7 @@ function schema:generateDirectiveMap()
           if argumentType == nil then
             error('Must supply type for argument "' .. name .. '" on "' .. directive.name .. '"')
           end
-          argumentType.defaultValue = argument.defaultValue
+          -- argumentType.defaultValue = argument.defaultValue
```

Comment:
Rebase on master after https://github.com/tarantool/graphql/pull/44 merge

Reply:
We didn't discussed #44 and made any decisions. So when it will be done I will rebase this PR on #44

Reply:
Fixed by merging with #44

Reply:
Since #44 is merged now, you can rebase